File: Components/WinForms/CS/TreeListMainDemo/Modules/About.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.Tutorials;


namespace DevExpress.XtraTreeList.Demos {
    public partial class About : ucOverviewPage {
        public About() {
            // This call is required by the Windows Form Designer.
            InitializeComponent();

            // TODO: Add any initialization after the InitializeComponent call
        }
        protected override string Line1Text {
            get {
                return "The XtraTreeList Suite";
            }
        }
        protected override string Line2Text {
            get {
                return "A hybrid treeview-listview control for the WinForms platform.";
            }
        }
        protected override string Line3Text {
            get {
                return "DevExpress Desktop Controls";
            }
        }
        protected override string Line4Text {
            get {
                return votedVSM;
            }
        }
    }
}

File: Components/WinForms/VB/TreeListMainDemo/Modules/About.vb
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.Tutorials


Namespace DevExpress.XtraTreeList.Demos
    Partial Public Class About
        Inherits ucOverviewPage
        Public Sub New()
            ' This call is required by the Windows Form Designer.
            InitializeComponent()

            ' TODO: Add any initialization after the InitializeComponent call
        End Sub
        Protected Overrides ReadOnly Property Line1Text As String
            Get
                Return "The XtraTreeList Suite"
            End Get
        End Property
        Protected Overrides ReadOnly Property Line2Text As String
            Get
                Return "A hybrid treeview-listview control for the WinForms platform."
            End Get
        End Property
        Protected Overrides ReadOnly Property Line3Text As String
            Get
                Return "DevExpress Desktop Controls"
            End Get
        End Property
        Protected Overrides ReadOnly Property Line4Text As String
            Get
                Return votedVSM
            End Get
        End Property
    End Class
End Namespace

